extension/chrome/settings/setup.ts

```diff
@@ -341,27 +450,35 @@ export class SetupView extends View {
       if (result.pubkeys.length) {
         const prvs = await KeyStoreUtil.parse(await KeyStore.getRequired(this.acctEmail));
         const parsedPubKeys = await KeyUtil.parseMany(result.pubkeys.join('\n'));
-        const hasMatchingKey = await asyncSome(prvs, (async (privateKey) => {
-          return parsedPubKeys.some((parsedPubKey) => privateKey.key.id === parsedPubKey.id);
-        }));
+        const hasMatchingKey = await asyncSome(prvs, async privateKey => {
+          return parsedPubKeys.some(parsedPubKey => privateKey.key.id === parsedPubKey.id);
+        });
         if (!hasMatchingKey) {
-          // eslint-disable-next-line max-len
-          throw new CompanyLdapKeyMismatchError(`Imported private key with ids ${prvs.map(prv => prv.key.id).join(', ')} does not match public keys on company LDAP server with ids ${parsedPubKeys.map(pub => pub.id).join(', ')} for ${this.acctEmail}. Please ask your help desk.`);
+          throw new CompanyLdapKeyMismatchError(
+            `Imported private key with ids ${prvs
+              .map(prv => prv.key.id)
+              .join(', ')} does not match public keys on company LDAP server with ids ${parsedPubKeys
+              .map(pub => pub.id)
+              .join(', ')} for ${this.acctEmail}. Please ask your help desk.`
+          );
```

Comment:
This formatting looks odd, the `.map` on next line doesn't look correct in the context of the `${}`

Reply:
Updated to
```js
const keyIds = prvs.map(prv => prv.key.id).join(', ');
const pubKeyIds = parsedPubKeys.map(pub => pub.id).join(', ');
throw new CompanyLdapKeyMismatchError(
  `Imported private key with ids ${keyIds} does not match public keys on company LDAP server with ids ${pubKeyIds} for ${this.acctEmail}. Please ask your help desk.`
);
```